src/_data/catalog/index.json

```diff
@@ -71,7 +71,7 @@
   },
   {
     "name": "Animation and Motion",
-    "description": "Bring animations to your app. Check out <a href=\"/animations/\">Animations</a> in Flutter for an overview.",
+    "description": "Bring animations to your app. Check out <a href=\"/development/ui/animations\">Animations</a> in Flutter for an overview.",
```

Comment:
Hmm, is this meant to be the page description? I don't think that we should have links in our descriptions.

Reply:
No, this was the source of great annoyance. It took me forever to find this broken link. I don't know who put this there but I'd be fine with removing it.